main/tests/server/src/com/google/refine/commands/column/AddColumnCommandTests.java

```diff
@@ -0,0 +1,61 @@
+
+package com.google.refine.commands.column;
+
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import com.google.refine.browsing.Engine;
+import com.google.refine.browsing.EngineConfig;
+import com.google.refine.commands.CommandTestBase;
+import com.google.refine.model.AbstractOperation;
+import com.google.refine.model.Project;
+import com.google.refine.operations.OnError;
+import com.google.refine.operations.column.ColumnAdditionOperation;
+
+public class AddColumnCommandTests extends CommandTestBase {
+
+    private AddColumnCommand command;
+    private Project project;
+    private EngineConfig engineConfig;
+
+    @BeforeMethod
+    public void setUpCommand() {
+        try {
+            setUpRequestResponse();
```

Comment:
I think you can get rid of this call to `setUpRequestResponse` since it's also annotated with `@BeforeMethod` in `CommandTestBase`. Can you try removing that and the try/catch block and see if the test still succeeds?